packages/tldraw/src/Tldraw.tsx

```diff
@@ -581,3 +644,17 @@ const StyledUI = styled('div', {
 const StyledSpacer = styled('div', {
   flexGrow: 2,
 })
+
+const getHeightAndWidthFromDataUrl = (
```

Comment:
Can this be part of ImageUtil or something?

Reply:
Surely